client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -37,34 +37,31 @@ const authUrl = '/wp-admin/admin.php?page=jetpack&connect_url_redirect=true&caly
 /**
  * Module variables
  */
-
-/***
- * Renders a header common to both the logged in and logged out forms
- * @param {String} siteUrl A site URL to display in the header
- * @param {Boolean} isConnected Is the connection complete
- * @returns {Object} The JSX for the form's header
- */
-const renderFormHeader = ( siteUrl, isConnected = false ) => {
-	const headerText = ( isConnected )
-		? i18n.translate( 'You are connected!' )
-		: i18n.translate( 'Connect your self-hosted WordPress' );
-	const subHeaderText = ( isConnected )
-		? i18n.translate( 'The power of WordPress.com is yours to command.' )
-		: i18n.translate( 'Jetpack would like to connect to your WordPress.com account' );
-	return(
-		<div>
-			<ConnectHeader headerText={ headerText }
-					subHeaderText={ subHeaderText } />
-			<CompactCard className="jetpack-connect__authorize-form-header">{ siteUrl }</CompactCard>
-		</div>
-	);
-};
-
 const JETPACK_CONNECT_TTL = 60 * 60 * 1000; // 1 Hour
 
 const LoggedOutForm = React.createClass( {
 	displayName: 'LoggedOutForm',
 
+	renderFormHeader( siteUrl, isConnected ) {
+		const headerText = ( isConnected )
+			? i18n.translate( 'You are connected!' )
+			: i18n.translate( 'Create a new account for Jetpack' );
+		const subHeaderText = ( isConnected )
+			? i18n.translate( 'The power of WordPress.com is yours to command.' )
```

Comment:
How do I trigger this string? Seems like I need to be connected somehow, and then get here?

I'd like to see it in context, because I think we need some better/different wording for it. It sounds a bit too dramatic :)